components/generic/interactions/components/interaction-users.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { memo } from 'react'
+
+import Avatar from '@components/user/avatar/avatar'
```

Comment:
Instead of creating a component for interaction users, make a generic component called "StackedAvatars" and reuse the same component for groups and user interactions